src/audio/google/google_rtc_audio_processing_mock.c

```diff
@@ -140,46 +138,40 @@ int GoogleRtcAudioProcessingReconfigure(GoogleRtcAudioProcessingState *const sta
 	return 0;
 }
 
-int GoogleRtcAudioProcessingProcessCapture_int16(GoogleRtcAudioProcessingState *const state,
-						 const int16_t *const src,
-						 int16_t *const dest)
```

Comment:
We can keep both: Both exists in the library.

Reply:
The library accepts both float32 between -1 and 1 and 16bits signed PCM. It uses another format internally so it would make sense to not do too much conversion outside and pass the data as straight forward as possible.